packages/block-library/src/quote/v2/edit.js

```diff
@@ -1,5 +1,94 @@
-function Edit() {
-	return <div>Quote block v2</div>;
-}
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import {
+	BlockControls,
+	RichText,
+	useBlockProps,
+	useInnerBlocksProps,
+	store as blockEditorStore,
+} from '@wordpress/block-editor';
+import {
+	BlockQuotation,
+	ToolbarGroup,
+	ToolbarButton,
+} from '@wordpress/components';
+import { useSelect } from '@wordpress/data';
+import { createBlock } from '@wordpress/blocks';
+
+export default function QuoteEdit( {
+	attributes: { attribution },
+	setAttributes,
+	isSelected,
+	insertBlocksAfter,
+	clientId,
+} ) {
+	const wrapperProps = useBlockProps();
+	const innerBlocksProps = useInnerBlocksProps(
+		{},
+		{
+			template: [ [ 'core/paragraph', {} ] ],
+			templateInsertUpdatesSelection: true,
+		}
+	);
+	const allBlockProps = useInnerBlocksProps( wrapperProps, {
+		template: [ [ 'core/paragraph', {} ] ],
+		templateInsertUpdatesSelection: true,
+	} );
```

Comment:
Why do we have two `useInnerBlocksProps` can't we just use one with a conditional first argument (hasAttribution)

Reply:
I thought it was a lot clearer this way. Can update it.

Reply:
It's fine by me :) I guess it's just a small preference.

Reply:
Thought thinking more, there might be a small perf impact by using useInnerBlockProps twice

Reply:
Updated it.